vitest.setup.ts

```diff
@@ -8,6 +8,8 @@ vi.mock('$app/stores', () => ({
 	page: mockPage
 }));
 
+vi.stubEnv('VITE_LOCAL_POUH_ISSUER_CANISTER_ID', 'qbw6f-caaaa-aaaah-qdcwa-cai');
```

Comment:
There is a race condition between importing the file that uses the env vars and stubbing the env vars.

To make sure that the env vars are stubbed we have two options:
* Stub in the test specifically and import everything dynamically, also breaking some mocks that depend on general import.
* Stub in the setup file (as is now). The disadvantage is that I can't change it for different tests scenarios. But I don't need to change this env var for different scenarios.

Reply:
Do we actually need to stub the env var? Can we specify it in the `.env.test`, would that work too?

Reply:
let me try

Reply:
Nice, it worked!

Reply:
Cool!